Tests/MapboxMapsTests/Location/LocationManagerTests.swift

```diff
@@ -37,4 +37,27 @@ internal class LocationManagerTests: XCTestCase {
         XCTAssertNil(locationManager.delegate)
         XCTAssertFalse(locationManager.showUserLocation)
     }
+
+    func testLocationManagerPuckUpdate() {
+        var locationOptions = LocationOptions()
+        locationOptions.showUserLocation = true
+        let locationManager = LocationManager(locationOptions: locationOptions,
+                                              locationSupportableMapView: locationSupportableMapMock)
```

Comment:
Similarly here — could we explicitly set a value for `puckType` rather than depending on its default value?

Reply:
Thanks, updated